js/ui/file_uploader.js

```diff
@@ -350,8 +351,41 @@ class FileUploader extends Editor {
             * @action
             */
 
+            /**
+            * @name dxFileUploaderOptions.uploadFile
+            * @type function
+            * @type_function_param1 file:File
+            * @type_function_param2 progressCallback(loadedBytes):any
+            * @type_function_return Promise<any>|any
+            */
             uploadFile: null,
+
+            /**
+            * @name dxFileUploaderOptions.uploadChunk
+            * @type function
+            * @type_function_param1 file:File
+            * @type_function_param2 chunksInfo:object
+            * @type_function_param2_field1 bytesLoaded:Number
```

Comment:
Maybe "bytes**Up**loaded"?

Reply:
"`bytesUploaded`" is better. But "`bytesLoaded`" has already been used in `onProgress` event.